lib/plugins/blocks.js

```diff
@@ -194,9 +196,8 @@ function inject (bot, { version }) {
 
   function blockAt (absolutePoint, extraInfos = true) {
     const loc = new Location(absolutePoint)
-    const key = columnKeyXZ(loc.chunkCorner.x, loc.chunkCorner.z)
 
-    const column = columns[key]
+    const column = bot.world.sync.getColumnAt(loc.chunkCorner.x, loc.chunkCorner.z)
```

Comment:
we should be able to use directly:
```js
const block = bot.world.sync.getBlock(absolutePoint)
```
also, I think that pworld should do the line:
```js
block.position = loc.floored
```

Reply:
yeah totally, I was mainly trying to make as little change as possible while making it works
I'll clean this up